package.json

```diff
@@ -45,6 +45,6 @@
   },
   "scripts": {
     "lint": "standard */*.js test/**/*.js",
-    "test": "npm run lint && tap --timeout=120 test/test-*"
```

Comment:
So, I'd really like this to stay for `npm t` because it's a good dev workflow. _If_ this is useful to separate out into CI maybe we need a separate `"ci"` target to use instead?

Reply:
Added `test:ci` because personally I find that having testing steps separated helps find root causes for failures easier.

Reply:
and reverted? was it too complicated?

Reply:
it broke windows tests because it added to variables to the environment. Rather go for a smaller changes than go changing things left and right.